sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/PropagateEmptyRelationSuite.scala

```diff
@@ -30,6 +30,7 @@ class PropagateEmptyRelationSuite extends PlanTest {
       Batch("PropagateEmptyRelation", Once,
         CombineUnions,
         ReplaceDistinctWithAggregate,
+        ReplaceExceptWithFilter,
```

Comment:
Why?

Reply:
If you think it is not useful here, i can remove it..

Reply:
@gatorsmile Please let me know if you think it's not useful here, also the
def replaceExceptWithFilter: Boolean = getConf(REPLACE_EXCEPT_WITH_FILTER) from the SQLConf, i'll push a new version with all the modifications, thanks.

Reply:
This rule is not related to `PropagateEmptyRelationSuite.scala`